optimize/upgrade/src/test/java/io/camunda/optimize/upgrade/migrate313to86/indices/EventIndexV4.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.optimize.upgrade.migrate313to86.indices;
+
+import io.camunda.optimize.service.db.schema.DefaultIndexMappingCreator;
+import java.io.IOException;
+import org.elasticsearch.xcontent.XContentBuilder;
+
+public class EventIndexV4 extends DefaultIndexMappingCreator<XContentBuilder> {
+
+  public static final int VERSION = 4;
+
+  @Override
+  public String getIndexName() {
+    return "event";
+  }
+
+  @Override
+  public String getIndexNameInitialSuffix() {
+    return "-000001";
+  }
+
+  @Override
+  public boolean isCreateFromTemplate() {
+    return true;
+  }
+
+  @Override
+  public int getVersion() {
+    return VERSION;
+  }
+
+  @Override
+  public XContentBuilder addProperties(final XContentBuilder xContentBuilder) throws IOException {
+    // only include sample field since this index is used for deletion IT
```

Comment:
what is meant by the sample field here? Why not simulate the real index properties?

Reply:
Because the migration is about deleting the entire index which I assumed is not affected by what its content looks like so I didnt want to spend time adding the fields manually (since all the constants are gone). Do you think the index fields are relevant for deletion? If they are I think I should add them

Reply:
Probably not relevant, I just generally lean towards testing as close to real behaviour as possible